packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -27,16 +27,17 @@ export const addNpmDependency: AsyncAction<{
   async ({ effects, actions, state }, { name, version, isDev }) => {
     effects.analytics.track('Add NPM Dependency');
     state.currentModal = null;
+    let newVersion = version;
 
-    if (!version) {
+    if (!newVersion) {
       const dependency = await effects.api.getDependency(name);
-      version = dependency.version;
+      newVersion = dependency.version;
     }
 
     await actions.editor.internal.addNpmDependencyToPackageJson({
       name,
-      version,
-      isDev,
+      version: newVersion,
+      isDev: !!isDev,
```

Comment:
Use `Boolean` instead of double negation

```suggestion
      isDev: Boolean(isDev),
```